manifest-base.json

```diff
@@ -10,24 +10,24 @@
   "title": "A32NX",
   "dependencies": [
     {
-      "package_version": "0.1.58",
+      "package_version": "0.1.62",
       "name": "asobo-aircraft-a320-neo"
     },
     {
-      "package_version": "0.1.52",
+      "package_version": "0.1.58",
       "name": "asobo-vcockpits-instruments-a320-neo"
     },
     {
-      "package_version": "0.1.53",
+      "package_version": "0.1.58",
       "name": "asobo-vcockpits-instruments-airliners"
     },
     {
-      "package_version": "0.1.66",
+      "package_version": "0.1.69",
       "name": "fs-base-aircraft-common"
     }
   ],
   "content_type": "AIRCRAFT",
-  "total_package_size": "00000000000221782906",
```

Comment:
you could remove this line entirely, build.py overrides it anyway.

Reply:
Roger that.